gcp_variant_transforms/pipeline_common.py

```diff
@@ -361,5 +370,90 @@ def create_output_table(full_table_id, total_base_pairs, schema_file_path):
   result = os.system(bq_command)
   if result != 0:
     raise ValueError(
-        'Failed to create a bigquery table using "{}" command.'.format(
+        'Failed to create a BigQuery table using "{}" command.'.format(
             bq_command))
+
+
+def delete_table(full_table_id):
+  bq_command = _BQ_DELETE_TABLE_COMMAND.format(FULL_TABLE_ID=full_table_id)
+  return os.system(bq_command)
+
+
+def delete_gcs_files(root_path):
+  gcs_command = _GCS_DELETE_FILES_COMMAND.format(ROOT_PATH=root_path)
+  return os.system(gcs_command)
+
+
+class LoadAvro(object):
+  def __init__(self,
+               avro_root_path,  # type: str
+               table_base_name,  # type: str
+               suffixes,  # type: List[str]
+               total_base_pairs  # type: List[int]
+              ):
+    assert len(suffixes) == len(total_base_pairs)
+
+    self._avro_root_path = avro_root_path
+    self._table_base_name = table_base_name.replace(':', '.')
+    self._suffixes = suffixes
+    self._total_base_pairs = total_base_pairs
+
+    self._num_load_jobs_retries = 0
+    self._suffixes_to_load_jobs = {}  # type: Dict[str, bigquery.job.LoadJob]
```

Comment:
It would be great in future to initiate load to BQ as soon as corresponding AVRO file has been created, instead of waiting for all AVRO files to be created. However, that seems like too much hassle right now. We should look into it later.